plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/edit/AltibaseQueueManager.java

```diff
@@ -0,0 +1,21 @@
+package org.jkiss.dbeaver.ext.altibase.edit;
```

Comment:
Copyright is required in all source files

Reply:
I put the Copyright :) If there is a way to check automatically, let me know and I will check before code commit.

Thanks for your notice.